.github/workflows/prompt-kit-profile-modality-review-repair.yml

```
@@ -0,0 +1,537 @@
+              count = text.count(old)
+              if count != expected:
+                  raise SystemExit(f'expected {expected} copies of {old!r}, observed {count}')
+              return text.replace(old, new)
+
+          # Canonical header markup: numeric navigation is retired. GNHF and Doctrine
+          # remain prompt content/classifications, while their former header slots become
+          # two independently persisted user profiles.
+          path = Path('build_prompt_kit.py')
+          text = path.read_text(encoding='utf-8')
+          old = '''    html.append('        <button class="cat-tab active" data-cat="all"><span class="tab-icon">&#128203;</span>All<span class="kbd">1</span></button>')
+    html.append('        <button class="cat-tab" data-cat="standard"><span class="tab-icon">&#128196;</span>Standard<span class="kbd">2</span></button>')
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)